CenterForOpenScience/osf.io
# [ENG-4283]Feature/remove registration contributor settings

## Purpose

The purpose of these changes is to remove the 'Settings' tab from contributor registrations.

## Changes

The code referencing the Settings tab for the project_header.mako file has been removed.

## QA Notes

Please make verification statements inspired by your code and what your code touches.

- Verify the Settings tab is removed in all project views
- Verify that user Settings can still be accessed via the navbar

What are the areas of risk?

Any concerns/considerations/questions that development raised?

There should be no area of risk, but sending out an informational email or notification that the tab has been removed from the project header and can be found on the main navbar would be helpful.

## Documentation

Help guides may need to reflect the update.

## Side Effects

There should be no side effects unless other routes invoke this one.

## Ticket

[](https://openscience.atlassian.net/browse/ENG-4283)
